quarkus-service-registry/README.adoc

```diff
@@ -0,0 +1,249 @@
++
+After you complete the topic setup, the new Kafka topic is listed in the topics table. Next, using the same configuration, create the `quotes` topic. You can now run the Quarkus application to start producing and consuming messages to and from these topics.
+
+.Verification
+ifdef::qs[]
+* Are the new Kafka topics `requests` and `quotes` listed in the topics table?
+endif::[]
+ifndef::qs[]
+* Verify that the new Kafka topics `requests` and `quotes` are listed in the topics table.
+endif::[]
+
+
+[id="proc-running-quarkus-registry-example-app_{context}"]
+== Running the Quarkus example application
+
+[role="_abstract"]
+After you configure your Quarkus application to connect to Kafka and  {registry} instances, and you create the Kafka topics, you can run the Quarkus application to start producing and consuming messages to and from these topics.
+
+The Quarkus project in this quick start consists of two processes:
+
+* The consumer application is implemented by the `QuotesResource` class. This class exposes the `/quotes/request` REST endpoint as an empty HTTP POST method. Every request to this endpoint is published to the `requests` topic. The same class exposes the `/quotes/quotes` REST endpoint that streams quotes from `quotes` topic. This application also has a minimal frontend that streams quotes from the latter endpoint using Server-Sent Events to the web page.
+* The producer application is implemented by the `QuotesProducer` class. This class consumes quote requests from `requests` topic and generates a random quote value that is published to the `quotes` topic.
+
+.Prerequisites
+* You've configured the Quarkus example application to connect to the Kafka and {registry} instances.
+* You've created the `requests` and `quotes` topics.
+
+.Procedure
+. On the command line, navigate to the `code-examples/quarkus-service-registry-quickstart` directory that you imported and run the consumer application.
++
+.Running the consumer example application
+[source]
+----
+$ cd ~/code-examples/quarkus-service-registry-quickstart/consumer
+$ ./mvnw quarkus:dev
+----
+. After the consumer application is running, in a web browser, go to http://localhost:8080/quotes.html[^] and verify that the application is available.
+
+. Leave the consumer application running, and run the producer application on a different terminal.
++
+.Running the producer example application
+[source]
+----
+$ cd ~/code-examples/quarkus-service-registry-quickstart/producer
+$ ./mvnw quarkus:dev
+----
+
+. When both the consumer and producer applications are running, view the generated quotes in the web browser at http://localhost:8080/quotes.html[^].
+
+. In the web console, go to *{registry}* > *{registry} Instances*,  select your {registry} instance, and view the automatically generated schemas for your applications.
```

Comment:
I would provide link to webconsole for convenience. 

For future: We will need to use UI/CLI to add extra permission in order for service account to be able to write schemas to registry.

Reply:
Good points, adding to prerequisites...